stat/init.lua

```diff
@@ -218,6 +219,27 @@ local function stat(args)
     stats['statperf.fibers'] = ed_cpu - st_cpu
     stats['statperf.fibers_t'] = ed_time - st_time
 
+    if detailed_replication then
+        stats['stat.detailed_replication'] = {
+            my_host = util.get_my_replica_host(),
+            replicaset = {},
+        }
+        for _, v in ipairs(box.info.replication) do
+            local item = {
+                uuid = v.uuid,
+                lsn = v.lsn,
+                host = stats['stat.detailed_replication']['my_host'],
```

Comment:
1. I think a comment like 'in master-slave topology the master has no `upstream` field in `box.info.replication`, so we need to get host:post information separatelly' would helpful here.